test/addons-napi/test_reference/test.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+// Flags: --expose-gc
+
+const common = require('../../common');
+const assert = require('assert');
+
+const test_reference = require(`./build/${common.buildType}/test_reference`);
+
+// This test script uses external values with finalizer callbacks
+// in order to track when values get garbage-collected. Each invocation
+// of a finalizer callback increments the finalizeCount property.
+assert.strictEqual(0, test_reference.finalizeCount);
```

Comment:
Fixed in https://github.com/nodejs/node/pull/12551/commits/caf86eddf3303bef65028745b9fd95107bb861df